runHarness/fixedConfigs.json

```diff
@@ -138,7 +138,7 @@
   "webServerCpus" : "275m",
   "webServerMem"  : "2Gi",
   "webServerImpl" : "nginx",
-  "nginxCacheVolumeSize" : "2Gi",
+  "nginxCacheVolumeSize" : "2150Mi",
 
```

Comment:
Its such a small change, but should a xsmall2 instead of xsmall be considered?

Reply:
Since it is such a small change, and since xsmall is so new that it is unlikely it has been used, I think that we should just keep it called xsmall.  It seems too confusing to add small2 for a change that won't affect performance.